lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -214,21 +214,21 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
     TEST_DATA = {
         # (providers, course_width, enable_ccx, view_as_ccx): # of sql queries, # of mongo queries, # of xblocks
-        ('no_overrides', 1, True, False): (48, 6, 13),
-        ('no_overrides', 2, True, False): (135, 6, 84),
-        ('no_overrides', 3, True, False): (480, 6, 335),
-        ('ccx', 1, True, False): (48, 6, 13),
-        ('ccx', 2, True, False): (135, 6, 84),
-        ('ccx', 3, True, False): (480, 6, 335),
+        ('no_overrides', 1, True, False): (49, 6, 13),
```

Comment:
I see from reading the other comments that this is caused by looking up the dark_lang model. It would be good to remove the extra call if possible and then revert all the changes to these tests.